### DISCUSSION TITLE
Elastic ingest pipelines not provided by SO

### DISCUSSION THREAD

**Discussion by rosswakelin** [Security-Onion-Solutions/securityonion#3410] [Unsupported Versions] [answered]
The "basic" version of Elastic 7.10 (or 7.11) comes with a large number of built-in ingest pipelines that might solve a lot of problems for a lot of SO users, such as Cisco switches and ASA, PA and Fortigate firewalls etc.  It there any reason why we cannot take those pipelines and try to add them to the Elastic version that SO uses?

**Comment by presianbg** [comment]
Hi,

You might want to check this in the docs:
https://docs.securityonion.net/en/2.3/elastic-features.html

Also by looking at the rest of the discussion, looks like there are few bugs with the Elastic Features.

**Comment by weslambert** [comment, marked as answer]
In the next update for Security Onion, we will be using Elastic with the Elastic license.  Because of this, we'll be able to more easily utilize things like the various Filebeat modules that are available.
